M2/Macaulay2/packages/TateOnProducts.m2

```diff
@@ -672,7 +672,7 @@ tateResolution(Module, List, List) := (M,low, high) ->(
     hi := apply(#regs, i->max(regs_i, high_i+1)); --hi
     N := presentation truncate(hi, M)**S^{hi};-- betti N
     Q := symExt(N,E); --betti Q
-    (res (coker Q,LengthLimit=>(sum hi-sum low)))**E^{hi}[sum hi]
+    (res (coker Q,LengthLimit=>(sum hi-sum low)))**E^{hi}[sum hi + 2 - length hi]
```

Comment:
@eisenbud @mikestillman -- is this change to your package okay?

Reply:
By the way, I have spoken to both David and Dan Erman about this before. Without it the Tate resolution for modules over products of three or more projective spaces will be incorrect and the [very last test](https://github.com/Macaulay2/M2/pull/1046/files#diff-057d56d65bb5d796a7b3c98fb2f88eddL231-L237) would fail, so if we don't hear back from them the short term solution would be to comment out that test again.

Reply:
@DanGrayson added a new commit to undo this change so the PR can be merged for now. If you hear back in time before the release, please revert the last commit.